server/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "opener http://localhost:3000 && node server.js"
```

Comment:
I love opener! Should this perhaps be in the other order in case it takes a bit to start on folks' computers to prevent them seeing an error?

ie. `node server.js && opener http://localhost:3000`